docs/modules/ROOT/pages/references/zones.adoc

```diff
@@ -1,47 +1,17 @@
 = APPUiO Cloud Zones
 
-== LPG 0
-
-WARNING: This is a test zone and could change again before going into production.
-
-____
-Direct link to the https://console.cloudscale-lpg-0.appuio.cloud/[OpenShift Console^]
-____
-
-Cloud Provider:: https://www.cloudscale.ch/[cloudscale.ch] / https://www.cloudscale.ch/en/api/v1#regions[Zone lpg1] / http://map.geo.admin.ch/?zoom=8&X=255362&Y=658250&crosshair=marker[Lupfig (AG)]
-OpenShift Version:: 4.8
-Kubernetes Version:: 1.21
-Kubernetes API endpoint:: https://api.cloudscale-lpg-0.appuio.cloud:6443
-Default App Domain:: apps.cloudscale-lpg-0.appuio.cloud
-Integrated Cluster Registry Domain:: registry.cloudscale-lpg-0.appuio.cloud
-SDN:: OVN-Kubernetes
-Storage::
-Supported storage classes:
-+
-* *ssd*: RWO (ext4 filesystem), https://www.cloudscale.ch/en/pricing#upgrade-options[cloudscale.ch SSD Storage]
-* *bulk*: RWO (ext4 filesystem), https://www.cloudscale.ch/en/pricing#upgrade-options[cloudscale.ch Bulk Storage]
-Outgoing IP:: 185.98.123.122
-CNAME:: cname.cloudscale-lpg-0.appuio.cloud
-
-== LPG 1
-
-WARNING: This is an experimental zone and could change again before going into production.
+== cloudscale.ch - LPG 2
 
 ____
-Direct link to the https://console.cloudscale-lpg-1.appuio.cloud/[OpenShift Console^]
+Direct link to the https://console.cloudscale-lpg-2.appuio.cloud/[OpenShift Console^]
 ____
 
 Cloud Provider:: https://www.cloudscale.ch/[cloudscale.ch] / https://www.cloudscale.ch/en/api/v1#regions[Zone lpg1] / http://map.geo.admin.ch/?zoom=8&X=255362&Y=658250&crosshair=marker[Lupfig (AG)]
-OpenShift Version:: 4.8
-Kubernetes Version:: 1.21
-Kubernetes API endpoint:: https://api.cloudscale-lpg-1.appuio.cloud:6443
-Default App Domain:: apps.cloudscale-lpg-1.appuio.cloud
-Integrated Cluster Registry Domain:: registry.cloudscale-lpg-1.appuio.cloud
-SDN:: Cilium
-Storage::
-Supported storage classes:
-+
-* *ssd*: RWO (ext4 filesystem), https://www.cloudscale.ch/en/pricing#upgrade-options[cloudscale.ch SSD Storage]
-* *bulk*: RWO (ext4 filesystem), https://www.cloudscale.ch/en/pricing#upgrade-options[cloudscale.ch Bulk Storage]
```

Comment:
No more listing storage classes?

Reply:
No, it's nowadays available in its own page: https://docs.appuio.cloud/user/explanation/storage-classes.html